server.js

```diff
@@ -92,19 +94,38 @@ const apiTracker = lib.setupTracker(openApiContents);
 
 // Detailed processes files
 
-const textileFile = "public/data/textile/processes_impacts.json";
-const foodFile = "public/data/food/processes_impacts.json";
+const textileImpactsFile = "public/data/textile/processes_impacts.json";
+const foodImpactsFile = "public/data/food/processes_impacts.json";
+const textileFile = "public/data/textile/processes.json";
+const foodFile = "public/data/food/processes.json";
+
+const processesImpacts = {
+  foodProcesses: JSON.parse(fs.readFileSync(foodImpactsFile, "utf8")),
+  textileProcesses: JSON.parse(fs.readFileSync(textileImpactsFile, "utf8")),
+};
 
 const processes = {
-  foodProcesses: fs.readFileSync(foodFile, "utf8"),
-  textileProcesses: fs.readFileSync(textileFile, "utf8"),
+  foodProcesses: JSON.parse(fs.readFileSync(foodFile, "utf8")),
+  textileProcesses: JSON.parse(fs.readFileSync(textileFile, "utf8")),
 };
```

Comment:
It really feels like it shouldn't be too hard to dynamically set all unwanted impacts to zero and get rid of the `processes.json` file.